projects/jdwp/tests/test_type_alias_definition.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from projects.jdwp.defs.schema import PrimitiveType
+from projects.jdwp.codegen.new_type_generator import get_type_alias_definition
+
+
+class TestTypeAliasDefinition(unittest.TestCase):
+    def test_type_alias_definitions(self):
+        for jdwp_type in PrimitiveType:
```

Comment:
Just assert that the `get_type_alias_definition` is what we expect for a few (two ?) types. We don't want to recompute definitions using similar logic in the test and in the tested code and then checking for equality.